app/lib/dartdoc/backend.dart

```diff
@@ -43,6 +46,60 @@ class DartdocBackend {
 
   DartdocBackend(this._db, this._storage);
 
+  /// Whether the storage bucket has a useable extracted data file.
+  /// Only the existence of the file is checked.
+  // TODO: decide whether we should re-generate the file after a certain age
```

Comment:
Is it redone when we bump pana or the runtime versions?

Reply:
Yes, it is redone.